modules/owncloud_web/pages/web_for_users.adoc

```diff
@@ -119,6 +107,57 @@ If you modified a document stored in Infinite Scale, you can always restore olde
 
 // FIXME: Elaborate more when it's working. Is there a diff to come? I hope so.
 
+== Sharing Roles and Permissions
+
+* Members can be added to a Space so that they gain access to all files and folders within the space. Members get the sharing *and* space permissions granted depending by their role.
+* Members can have different roles and permissions in different spaces.
+* When sharing single files and folders only, sharees are granted the sharing, *but not* the space permissions.
+
+The following table describes the default roles and it's permissions.
+
+NOTE: Permissions *sum up* as higher the role gets.
+
+[width=100,cols=".^30%,.^100%,.^100%",options="header",]
+|===
+| Role
+^| Sharing Permissions
+^| Space Permissions
+
+| Viewer
+a| * read documents,
+* view images,
+* play videos.
+a| * get info about the space.
+
+| Editor
+a| * modify documents
+* delete documents,
+* upload content to a space.
+a| * edit the space name,
+* edit the space description,
+* add or change the image for the space,
+* empty the spaces trash bin,
+* recover files from the spaces trashbin.
+
+| Manager
+|
+a| * invite additional members,
+* share a space,
+* deactivate and activate a space ^(1)^,
+* delete a space,
+* change the roles of members, +
+including those of other managers. A downgrade to viewer or editor is possible.
+
+|
+^a| Personal Sharing View +
+image:web-for-users/sharing-menu.png[Personal Sharing Menu,width=300] +
+Spaces Sharing View +
+image:web-for-users/spaces-share-menu.png[Spaces Sharing Menu,width=300]
+^| image:web-for-users/spaces-menu.png[Spaces Menu,width=300]
+|===
+
+(1) ... With deactivating, a Manager can disable any access to a space including sharing and syncing. This can be used to keep data in a frozen state before finally deleting it later on. To regain full access including sharing and syncing, a deactivated space can be activated again.
```

Comment:
```suggestion
(1) ... By deactivating, a Manager can disable any access to a space including sharing and syncing. This can be used to keep data in a frozen state before finally deleting it later on. To regain full access including sharing and syncing, a deactivated space can be activated again.
```